src/templates.rs

```diff
@@ -12,10 +12,11 @@ where
         match self.0.render() {
             Ok(html) => Html(html).into_response(),
             Err(error) => {
-                tracing::error!("Failed to render template: {error:?}");
+                let message = format!("Failed to render template: {error:?}");
+                tracing::error!(message);
```

Comment:
`tracing::error!("{message}")`, otherwise this would be unnecessarily rendered as `message=<message>`.